tests/src/sanity.rs

```diff
@@ -662,9 +662,34 @@ mod tests {
 
         let mut args = vec!["--enable-fs"];
 
-        if let Some(ephemeral_flag) = agent.flag() {
```

Comment:
no need for ephemeral_flag for this test anymore?

Reply:
we don't test ephemeral on macOS as it's redundant and it always tested job only so redundant